llvm/unittests/CodeGen/SelectionDAGPatternMatchTest.cpp

```diff
@@ -263,6 +268,9 @@ TEST_F(SelectionDAGPatternMatchTest, matchUnaryOp) {
   EXPECT_FALSE(sd_match(Sub, m_Neg(m_Value())));
   EXPECT_FALSE(sd_match(Neg, m_Not(m_Value())));
   EXPECT_TRUE(sd_match(VScale, m_VScale(m_Value())));
+
+  EXPECT_TRUE(sd_match(FPToUI, m_FPToUI(m_Value())));
+  EXPECT_TRUE(sd_match(FPToSI, m_FPToSI(m_Value())));
```

Comment:
Add a fail test as well, maybe matching the fptoui to FPToSi?

Reply:
Thank you. Added.